internal/resolution/resolve.go

```diff
@@ -104,6 +114,92 @@ func Resolve(ctx context.Context, cl client.ResolutionClient, m manifest.Manifes
 	return result, nil
 }
 
+func resolvePostProcess(ctx context.Context, cl client.ResolutionClient, m manifest.Manifest, opts ResolveOpts, graph *resolve.Graph) (*resolve.Graph, error) {
+	if m.System() == resolve.Maven && opts.MavenManagement {
+		// Add a node & edge for each dependency in dependencyManagement that doesn't already appear in the resolved graph
+		manifestSpecific, ok := m.EcosystemSpecific.(manifest.MavenManifestSpecific)
+		if !ok {
+			return graph, errors.New("invalid MavenManifestSpecific data")
+		}
+
+		// Search through OriginalRequirements management dependencies in this pom only (not parents).
+		for _, req := range manifestSpecific.OriginalRequirements {
```

Comment:
maybe we can refactor `manifestSpecific.RequirementsForUpdates` for this purpose?

Reply:
Possibly, but it seems like a bigger change and I don't know how it would impact the maven suggester

Reply:
fair - maybe a TODO to explore later (also I don't know if we still want to continue working on `osv-scanner update`)